src/qt/walletframe.cpp

```diff
@@ -73,6 +74,10 @@ bool WalletFrame::addWallet(WalletModel *walletModel)
     connect(walletView, &WalletView::encryptionStatusChanged, gui, &BitcoinGUI::updateWalletStatus);
     connect(walletView, &WalletView::incomingTransaction, gui, &BitcoinGUI::incomingTransaction);
     connect(walletView, &WalletView::hdEnabledStatusChanged, gui, &BitcoinGUI::updateWalletStatus);
+    connect(gui, &BitcoinGUI::setPrivacy, walletView, &WalletView::setPrivacy);
+    QTimer::singleShot(0, walletView, [this, walletView] {
```

Comment:
Why singleShot? Anyway, if you follow the OptionsModel then this is not needed.

Reply:
> Why singleShot?

To apply the privacy mode to a just opened wallet.

Reply:
But this is the same as just calling `walletView->setPrivacy(gui->isPrivacyModeActivated())`.

Reply:
> But this is the same as just calling `walletView->setPrivacy(gui->isPrivacyModeActivated())`.

Indeed :)
Thank you!

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16432#issuecomment-633592495).